### ISSUE TITLE
Password reset does not work out of the box.

### ISSUE THREAD

**Issue by Artem-Schander** [bitExpert/magento2-force-login#118]
Hi guys,
firstly thanks for the cool package. Makes the life a bit easier.

It took me awhile till I got the reason for not being able to reset a password. The whitelist entry for `/customer/account/resetpasswordpost` is not present by default. I think it should be.
I'll try to take the time and get together a pull request in the next days.
Please let me know if there are reasons why it's not there. It would be nice if I could avoid doing unnecessary work :)

**Comment by shochdoerfer** [comment]
Good catch, thanks. This is not missing for a particular reason, just a minor glitch ;) We'd love to get a PR from you to fix this.
